.github/workflows/kind.yaml

```diff
@@ -24,6 +24,14 @@ jobs:
   installation-and-connectivity:
     name: Kind Installation and Connectivity Test
     runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # run connectivity tests explicitly without concurrency
+          - test-concurrency: 1
```

Comment:
Would it make sense to retain fail-fast for test-concurrency: 1 for quicker iteration and keep it off for higher concurrency?

Reply:
Good catch! I've disabled `fail-fast` for the debugging purpose.
I think it's better to remove it at all for the workflow.